doc/classes/ColorPicker.xml

```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		Displays a color picker widget. Useful for selecting a color from an RGB/RGBA colorspace.
+		Note: You can use a [ColorPickerButton] to display a [ColorPicker] in a pop-up.
```

Comment:
```suggestion
		[b]Note:[/b] This control is the color picker widget itself. You can use a [ColorPickerButton] instead if you need a button that brings up a [ColorPicker] in a pop-up.
```

Reply:
I'm fine with this fix and *could* use the "Commit suggestion" button via web UI but I assume that's going to turn this a multiple commit "thing" so probably best for someone with maintainer access just to make the change instead.

Reply:
If you want to contribute to the main repo of Godot, you need to be able to squash and rebase, yes. We can close this PR as salvageable, if you can't finish it.